lab-kyle/package.json

```diff
@@ -0,0 +1,23 @@
+{
```

Comment:
This is a good start, there are a few ways you can improve the file.  Please see my comments below for more information.